oneview-ansible.md

```diff
@@ -3504,7 +3504,7 @@ Manage OneView Ethernet Network resources.
     hostname: 172.16.101.48
     username: administrator
     password: my_password
-    api_version: 1200
+    api_version: 800
```

Comment:
Can you please check this file? Its documenting for Ethernet networks instead of Storage pools

Reply:
This might have come from rebase. I haven't updated it.